src/Marten/Linq/QueryHandlers/JsonQueryHandler.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Data.Common;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Baseline;
+using Marten.Schema;
+using Marten.Services;
+using Marten.Util;
+using Npgsql;
+using Remotion.Linq;
+
+namespace Marten.Linq.QueryHandlers
+{
+    public class JsonQueryHandler : IQueryHandler<string>
+    {
+        private readonly IDocumentSchema _schema;
+        private readonly QueryModel _query;
+        private readonly ISelector<string> _selector;
+        private readonly IDocumentMapping _mapping;
+
+        public JsonQueryHandler(IDocumentSchema schema, QueryModel query)
+        {
+            _mapping = schema.MappingFor(query);
+            _schema = schema;
+            _query = query;
+
+            var selector = _schema.BuildSelector<string>(_mapping, _query);
+
+            _selector = selector;
+        }
+
+
+        public Type SourceType => _query.SourceType();
+        public void ConfigureCommand(NpgsqlCommand command)
```

Comment:
I tried to use ListQueryHandler but couldn't since I was returning a `string` rather than a `List<string>` which made me have to do that.